tests/rhoai_mcp/conftest.py

```
@@ -292,7 +294,131 @@ def rhoai_mcp_ready(
+    teardown_resources: bool,
+) -> Generator[Namespace, Any, Any]:
+    """Namespace for model deployment via MCP tools."""
+    with create_ns(
+        admin_client=admin_client,
+        name=RHOAI_MCP_MODEL_DEPLOY_NAMESPACE,
+        teardown=teardown_resources,
+    ) as ns:
+        yield ns
+
+
+# Model deployment test persona – wider than the RBAC test personas (rbac_reader/rbac_deployer)
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)